service/ui/page/profile/edit-success.html

```diff
@@ -0,0 +1,17 @@
+<Page title="Edit Profile Complete">
```

Comment:
add :user here, so the sign out link displays properly.

Reply:
Updated to make current user available globally so we don't have to pass around.